flow-typed/box-ui-elements.js

```diff
@@ -733,6 +733,8 @@ type AppActivityItems = {
 type FeedItem = Comment | Task | TaskNew | BoxItemVersion | AppActivityItem;
 type FeedItems = Array<FeedItem>;
 
+type FeedItemType = 'task' | 'comment';
```

Comment:
We've been trying to avoid adding types to this file, as it's globally imported to all projects. Can it live as an explicit import?